include/openssl/err.h

```diff
@@ -163,12 +163,25 @@ struct err_state_st {
 #  define X509err(f, r) ERR_raise_data(ERR_LIB_X509, (r), NULL)
 # endif
 
+/*
+ * Error codes are packed into a single 32-bit number, and have three parts,
+ * library (L), function (F, not used as of 3.0), and reason (R). It is
+ * split as follows:
+ *      LL FFF RRR
+ * The top two bits of LL are zero and fatal-error-flag.
```

Comment:
bits => nibbles

Reply:
The fatal-error-flag is part of the RRR (somewhere in the middle)...  and is never used.  It actually looks like an unfinished idea that just hung around for eternity.

Reply:
You're right, 0x40, not shifted.  So reason codes are limited to 0x3FF.  I rewrote the comment and am pushing a fixup commit.  Should I remove/deprecate ```ERR_FLAG_FATAL``` or just leave it?